<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StartWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>0</y>
      <width>421</width>
      <height>82</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>26</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Scheduling Simulator</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>71</y>
      <width>423</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>调度算法</string>
    </property>
    <widget class="QRadioButton" name="radioButton_non">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>149</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>优先权调度-非抢占式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_preemptive">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>60</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>优先权调度-抢占式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_rr">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>90</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>时间片轮转调度</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方-简</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>210</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方-简</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>320</width>
      <height>205</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/res/gallery.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>label_image</zorder>
   <zorder>label_title</zorder>
   <zorder>pushButton_start</zorder>
   <zorder>pushButton_exit</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
